packages/agoric-cli/src/commands/vaults.js

```diff
@@ -59,6 +77,116 @@ export const makeVaultsCommand = async logger => {
       }
     });
 
+  vaults
+    .command('info')
+    .description('show governance parameters of a vault type')
+    .requiredOption('--manager [N]', 'manager number', s => Number(s), 0)
```

Comment:
not necessary but could be nice to query factory params when the manager arg is omitted